Core/Core/Planner/CreateTodoViewController.swift

```diff
@@ -93,28 +93,11 @@ public class CreateTodoViewController: UIViewController, ErrorViewController {
     }
 
     @IBAction func showDatePicker(_ sender: Any) {
-        datePicker.preferredDatePickerStyle = .wheels
-        datePicker.datePickerMode = .dateAndTime
-        let toolbar = UIToolbar()
-        toolbar.sizeToFit()
-        let done = UIBarButtonItem(title: NSLocalizedString("Done", bundle: .core, comment: ""), style: .plain, target: self, action: #selector(didPickDate))
-        let space = UIBarButtonItem(barButtonSystemItem: .flexibleSpace, target: nil, action: nil)
-        let cancel = UIBarButtonItem(title: NSLocalizedString("Cancel", bundle: .core, comment: ""), style: .plain, target: self, action: #selector(cancelDatePicker))
-        toolbar.setItems([cancel, space, done], animated: false)
-
-        dateTextField.inputAccessoryView = toolbar
-        dateTextField.inputView = datePicker
-        dateTextField.becomeFirstResponder()
-    }
-
-    @objc func didPickDate() {
-        dateTextField.resignFirstResponder()
-        selectedDate = datePicker.date
-        dateTextField.text = formattedDate
-    }
-
-    @objc func cancelDatePicker() {
-        dateTextField.resignFirstResponder()
+        let storyboard = UIStoryboard(name: "Sheet", bundle: .core)
```

Comment:
There's a `ViewControllerLoader` helper to load viewcontrollers from storyboards, just name the storyboard the same as the viewcontroller and use that.